liferay/liferay-npm-tools
# feat(preset): add shared @clayui and react packages

I'm sending over here an update on the preset that contains:
- New shared `@clayui/*` packages.
- New shared react and react utilities packages.

The version range that's in here just reflects the current version that's being installed in portal, so we assume we will always have at least that one or a bigger version running in portal.

**test plan** 
- Take https://github.com/andresfulla/liferay-portal/pull/95 and remove the additional `.npmbundlerrc` configuration
- Update `npmscripts` to use the newer preset
- Build both `segments-web` and `frontend-taglib-clay` modules and verify that:
    - `segments-web` is not packaging `@clayui/*` nor `react*` packages in its classes folder
    - `frontend-taglib-clay` does not contain `react*` packages in its classes folder
